api/app/Http/Requests/Api/EventType/EventTypeRequest.php

```diff
@@ -35,6 +36,8 @@ public function rules(): array
                 'string',
                 'date_format:Y-m-d H:i:s'
             ],
+            "location_type" => Rule::in(LocationTypes::getAllLocationTypes()),
+            "coordinates" => "json"
```

Comment:
We should work with coordinates like with any other field, but store it as `json`.
So here it will be an array with `lng` ant `lat` fields.